### PR TITLE
Task/minor tweaks aggregate report ui

### PR DESCRIPTION
**Pull request by wtritch**

### CODE REVIEW

File: webapp/src/main/webapp/src/app/aggregate-report/results/aggregate-report.component.ts [hunk 1]
```diff
@@ -38,8 +38,8 @@ export class AggregateReportComponent implements OnInit, OnDestroy {
     this.assessmentDefinitionsByAssessmentTypeCode = this.route.snapshot.data[ 'assessmentDefinitionsByAssessmentTypeCode' ];
     this.options = this.route.parent.snapshot.data[ 'options' ];
     this.report = this.route.snapshot.data[ 'report' ];
-    this.reportSizeSupported = !Utils.isUndefined(this.report.metadata.row_count)
-      && (Number.parseInt(this.report.metadata.row_count) < SupportedRowCount);
+    this.reportSizeSupported = !Utils.isUndefined(this.report.metadata.totalCount)
+      && (Number.parseInt(this.report.metadata.totalCount) < SupportedRowCount);
```
Comment (wtritch) [inline webapp/src/main/webapp/src/app/aggregate-report/results/aggregate-report.component.ts]: I named the metadata property for aggregate-report row counts "totalCount"

File: webapp/src/main/webapp/src/app/aggregate-report/results/aggregate-report.component.ts [hunk 2]
```diff
@@ -87,7 +87,7 @@ export class AggregateReportComponent implements OnInit, OnDestroy {
   private initializeReportTables(rows: AggregateReportRow[]): void {
 
     const subjects = this.options.subjects;
-    const assessmentDefinition = this.assessmentDefinitionsByAssessmentTypeCode.get(this.report.request.assessmentTypeCode);
+    const assessmentDefinition = this.assessmentDefinitionsByAssessmentTypeCode.get(this.report.request.reportQuery.assessmentTypeCode);
```
Comment (wtritch) [inline webapp/src/main/webapp/src/app/aggregate-report/results/aggregate-report.component.ts]: The report.request payload is the full AbstractExamExportRequest, not just the ReportQuery in the case of an aggregate report request.

File: webapp/src/main/webapp/src/app/report/report.service.ts [hunk 3]
```diff
@@ -130,7 +130,7 @@ export class ReportService {
       headers: new Headers({
         'Accept': 'application/json',
       }),
-      responseType: ResponseContentType.Blob
+      responseType: ResponseContentType.Json
```
Comment (wtritch) [inline webapp/src/main/webapp/src/app/report/report.service.ts]: The front-end wasn't parsing the response as JSON when we told it to use Blob... I suppose that makes sense.

Comment (esotrope) [inline webapp/src/main/webapp/src/app/report/report.service.ts]: ah good catch - think this is default actually - probably can remove the responseType param all together

File: webapp/src/main/webapp/src/app/report/report.service.ts [hunk 4]
```diff
@@ -204,6 +204,7 @@ export class ReportService {
     local.subjectCode = remote.subjectCode;
     local.schoolYear = remote.schoolYear;
     local.metadata = remote.metadata || {};
+    local.request = remote.request;
```
Comment (wtritch) [inline webapp/src/main/webapp/src/app/report/report.service.ts]: Stuff the returned request into the Report front-end response object.

**Pull request merged**